.gitignore

```diff
@@ -244,3 +244,8 @@ video_creation/data/envvars.txt
 
 config.toml
 video_creation/data/videos.json
```

Comment:
as you can see the videos.json is in the gitignore....

Reply:
yeah that is strange